Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 pipeline {
-    agent any
+    agent { label 'vault-token' }
```

Comment:
@benjamincarlin here's (one example of) how to restrict where Jenkins jobs run. I had to add a new label to the worker node.